backend/__fixtures__/shoots.js

```diff
@@ -68,7 +69,11 @@ function getShoot ({
   secretBindingName = 'foo-secret',
   seed = 'infra1-seed',
   hibernation = { enabled: false },
-  kubernetesVersion = '1.16.0'
+  kubernetesVersion = '1.16.0',
+  advertisedAddresses = [{
```

Comment:
Do we really want that every shoot in the tests gets `advertisedAddresses` even in cases were this is not releavant. And the default name does not match the shootname and projectname. This is very confusing to me.

Reply:
✅